src/scripts/devhub.zig

```diff
@@ -41,10 +41,11 @@ fn devhub_coverage(shell: *Shell) !void {
     try shell.project_root.makePath("./src/devhub/coverage");
 
     const kcov: []const []const u8 = &.{ "kcov", "--include-path=./src", "./src/devhub/coverage" };
-    try shell.exec("{kcov} ./zig-out/bin/test", .{ .kcov = kcov });
-    try shell.exec("{kcov} ./zig-out/bin/fuzz lsm_tree 92", .{ .kcov = kcov });
-    try shell.exec("{kcov} ./zig-out/bin/fuzz lsm_forest 92", .{ .kcov = kcov });
-    try shell.exec("{kcov} ./zig-out/bin/vopr 92", .{ .kcov = kcov });
+    const kcov_args = .{ .kcov = kcov };
+    try shell.exec("{kcov} ./zig-out/bin/test", kcov_args);
+    try shell.exec("{kcov} ./zig-out/bin/fuzz --events-max=500000 lsm_tree 92", kcov_args);
+    try shell.exec("{kcov} ./zig-out/bin/fuzz --events-max=500000 lsm_forest 92", kcov_args);
+    try shell.exec("{kcov} ./zig-out/bin/vopr 92", kcov_args);
```

Comment:
If we want to have some deduce here, might flip this around and write a for-loop?